orchagent/portsorch.cpp

```diff
@@ -61,6 +61,7 @@ extern string gMyAsicName;
 #define DEFAULT_VLAN_ID     1
 #define MAX_VALID_VLAN_ID   4094
 
+#define PORT_STAT_POLLING_SEC                             1
```

Comment:
Why do we need an agressive 1 second timer? Can this be something like 5 or 10 seconds if it is for show command purpose alone?

Reply:
Sure, no problem